File: WorldFeed/Server/src/Services/Trend/WorldFeed.Trend.API/Application/Parameters/GetTrendsLocationParameters.cs
namespace WorldFeed.Trend.Application.Parameters
{
    using WorldFeed.Common.Public.Parameters;

    /// <summary>
    /// https://developer.twitter.com/en/docs/trends/locations-with-trending-topics/api-reference/get-trends-available
    /// </summary>
    public interface IGetTrendsLocationParameters : ICustomRequestParameters
    {
    }

    /// <inheritdoc />
    public class GetTrendsLocationParameters : CustomRequestParameters, IGetTrendsLocationParameters
    {
    }
}

File: Sharebook/Client/src/app/core/Public/Parameters/TrendsClient/GetTrendsLocationParameters.ts
import {CustomRequestParameters, ICustomRequestParameters} from "../CustomRequestParameters";

// https://developer.twitter.com/en/docs/trends/locations-with-trending-topics/api-reference/get-trends-available
export interface IGetTrendsLocationParameters extends ICustomRequestParameters {
}

export class GetTrendsLocationParameters extends CustomRequestParameters implements IGetTrendsLocationParameters {
}
